### DISCUSSION TITLE
Inventory please!

### DISCUSSION THREAD

**Discussion by Raddish2** [Ladysnake/Requiem#329] [Q&A] [answered]
It would be awesome if there were a way to toggle the inventory function for all mobs. I'm doing a playthrough with origins, requiem, and the pandemonium mod. It's really annoying that I cant use my inventory when I possess passive mobs, I thought it would be really cool to do the whole playthrough using a fox or a bee as my main body but it's impossible because I can't use my inventory.

**Comment by Raddish2** [comment]
Also, if there is some way to do this already please let me know!

**Reply by Pyrofab** [reply, marked as answer]
You can use datapacks to add inventories to all mobs: https://ladysnake.github.io/wiki/requiem/configuration
